api/views.py

```diff
@@ -145,7 +145,8 @@ def validate(self, data):
 
     def _postsave(self, validated_data, instance):
         instance = super(CardSerializer, self)._postsave(validated_data, instance)
-        instance.force_cache_member()
+        instance.update_cache("member")
+        instance.update_cache("cameos")
```

Comment:
Did you test that it works without the `force_update_cache`? I can't remember why I used force in the first place.

Reply:
yep, changing the member still works.